### DISCUSSION TITLE
Order by element exists in array

### DISCUSSION THREAD

**Discussion by TheMadKow** [ravendb/ravendb#14086] [Q&A]
Hi,
I'm trying to achieve a query where I want to order the results by id exits in an array.

Example (C#) : 
---------------
1. Let's have an index where one of the fields is 
public string[] HighForUserId ;

2. On the query side I want to do something like 
query.orderBy(x => x.HighForUserId.any(y => y == currentUserId));

So, Is it possible ?
My other alternative is by doing 2 queries, using where (true) and where (false) for this condition and joining them after the fact.

Best,
Eliran

**Comment by ayende** [comment]
You can do that using something like:

```
from Items 
where Tag = $tag and (boost(HighForUserId = $currentUserId), 10) or true)
order by score()
```

This will ensure that documents with a match would score higher and be first.
